### ISSUE TITLE
I get a "not found" error using useStore.

### ISSUE THREAD

**Issue by ed2910** [jhonnymichel/react-hookstore#43]
Hi there, this looks so promising and your examples work perfectly, thank you. 

I understand you're dealing with some advanced issues (the open threads) but this is a simple newbie issue:

I'm having reference issues.
In your "basic" example, both components are in the **same** file. In my app, I have two **separate** components not in the same file, a parent and a child. In the parent, I create the store and it's all good to display a counter. 

But in the child, I get an "uncaught store with name ... does not exist error" when I add the useStore line. 

I just can't figure out how to reference the store created in the parent? (I tried getStoreByName, same problem). 

Thanks,
Ed

 

Thanks,
Ed

**Comment by jhonnymichel** [comment]
Hello!

I followed your steps here: https://codesandbox.io/s/sad-sky-qsh8c?file=/src/App.js

and as you can see its working fine.

what you described should not be happening. Could you provide a working example so I can investigate?

**Comment by ed2910** [comment]
Hi,

Thanks so much for your prompt response. I replicated your example into my environment , and now get a clearer error message:

"Failed to compile
./src/views/resumes/ResumeCreateView/testHookStore.js
  Line 8:21:  React Hook "useStore" is called in function "testHookStore" which is neither a React function component or a custom React Hook function  react-hooks/rules-of-hooks.

I googled but it's just not clear to my why my syntax is making testHookStore  not appear as a function component.

The files are attached in a zip. 

Regards,
Edwin
[ResumeCreateView.zip](https://github.com/jhonnymichel/react-hookstore/files/4678431/ResumeCreateView.zip)
